tests/integration/test_users.py

```diff
@@ -85,7 +85,7 @@ def test_get_user_by_id(client, auth_headers, admin_user):
 
 def test_get_user_by_property_manager_id(client, auth_headers, new_property):
```

Comment:
This test seems redundant, and is possibly more focused on implementation rather than behavior. 
Finding a resource by an id should already be under test. I think it is safe to delete this test.

Reply:
Yes. I can remove this test